geomstats/_backend/pytorch/linalg.py

```diff
@@ -3,19 +3,21 @@
 import numpy as np
 import scipy.linalg
 import torch
+from . import logm
```

Comment:
You do not need this line nor line 13, if you define `logm` in this file.

Could you define `logm` in this file, by importing the code from `logm.py` here?